devito/dse/transformer.py

```diff
@@ -16,6 +16,8 @@
 }
 """The DSE transformation modes."""
 
+MAX_SKEW_FACTOR = 8
+configuration.add('skew_factor', 0, range(MAX_SKEW_FACTOR)) 
```

Comment:
I tried to move it, but did not succeed yet. Where do you think should this live in?